cmd/add.go

```diff
@@ -24,8 +24,11 @@ type tomlConfig struct {
 }
 
 type profile struct {
-	Email    string `toml:"email"`
-	AuthType string `toml:"auth_type"`
+	Email              string              `toml:"email"`
+	AuthType           string              `toml:"auth_type"`
+	SessionDuration    int                 `toml:"session_duration"`
```

Comment:
this would be nice as a `time.Duration` value of something like "1h3m0s" too but may not be needed

Reply:
added in 8481dd4